pkg/agent/wireguard/client_linux.go

```diff
@@ -84,14 +87,16 @@ func New(clientSet clientset.Interface, nodeConfig *config.NodeConfig, wireGuard
 		wgClient:                wgClient,
 		nodeName:                nodeConfig.Name,
 		k8sClient:               clientSet,
+		mcClient:                mcClientSet,
+		mcNamespace:             mcNamespace,
 		wireGuardConfig:         wireGuardConfig,
 		peerPublicKeyByNodeName: &sync.Map{},
 		gatewayConfig:           nodeConfig.GatewayConfig,
 	}
 	return c, nil
 }
 
-func (client *client) Init() error {
+func (client *client) Init(multiclusterEnabled bool) error {
```

Comment:
Probably rename `multiclusterEnabled` to `isMulticluster`. It is not about "enabling" multi-cluster.

Reply:
Done.